tools/svd2regs/svd2regs.py

```diff
@@ -386,6 +386,12 @@ def main():
     code = generate(*parse(args.peripheral, args.mcu, args.svd, args.group))
     if args.fmt is not None:
         code = rustfmt(code, args.path, *args.fmt.strip("'").split(" "))
+    header = "//" * 40 + "\n"
+    header += "// Autogenerated by svd2regs.py on " + str(datetime.date.today()) + "\n"
+    header += "//   " + ' '.join(sys.argv) + '\n'
```

Comment:
I know it doesn't matter, but you used `"\n"` on one line and `'\n'` on the other.

Reply:
fixed